plugins/modules/vmware_guest_controller.py

```diff
@@ -207,7 +215,8 @@
                 ],
                 "controller_label": "SCSI controller 0",
                 "controller_summary": "LSI Logic SAS",
-                "controller_unitnumber": 3
+                "controller_unitnumber": 3,
+                "controller_sharedbus": 'noSharing'
```

Comment:
```suggestion
                "bus_sharing": 'noSharing'
```

If I see this correct, the parameter is `bus_sharing` and not `controller_sharedbus`.

Reply:
The example references what is returned by `gather_disk_controller_facts` (line 351)
Which has `controller_sharedbus`, not `bus_sharing`

Didfind a typo, tho. `ontroller_sharedbus`

Reply:
Sorry, I didn't realize that this is an example of the return value. You're right, it's correct. Although the return value feels kind of wrong... I'll comment on this line.